plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1617,6 +1617,7 @@ modelement.projectile=Projectile
 modelement.recipe=Recipe
 modelement.structure=Structure
 modelement.tab=Creative tab
+modelement.tag=Tag
```

Comment:
Tags are no longer mod elements but tag elements, so this should probably be fixed by changing the key used, not by re-adding the old one.

Reply:
It would make sense to use a different key, I agree

Reply:
What should the new key be?

Reply:
There is `workspace.tags.tag_name`, if it doesn't match, it can be `workspace.tags.tag_name_validator`.

Reply:
Probably "workspace.tags.tag"